bundles/org.eclipse.ui.forms/src/org/eclipse/ui/forms/widgets/ExpandableComposite.java

```diff
@@ -887,15 +887,47 @@ public int getExpansionStyle() {
 	/**
 	 * Programmatically changes expanded state.
 	 *
+	 * Equivalent to <code>setExpanded(expanded, false)</code>. @see {@link #setExpanded(boolean, boolean)}.
+	 *
 	 * @param expanded
 	 *            the new expanded state
 	 */
 	public void setExpanded(boolean expanded) {
-		internalSetExpanded(expanded);
+		setExpanded(expanded, false);
+	}
+
+	/**
+	 * Programmatically changes expanded state.
+	 *
+	 * @param expanded
+	 *            the new expanded state
+	 * @param fireExpanding
+	 *            if true, listeners will be fired
+	 */
+	public void setExpanded(boolean expanded, boolean fireExpanding) {
```

Comment:
As CI says, new API require to bump the bundle minor version (if not already done since last release) and add a `@since x.y.z` annotation in javadoc.

Reply:
done